README.md

```diff
@@ -1 +1,149 @@
+
+```bash
+# Scala Stream Collector image
+docker pull snowplow-docker-registry.bintray.io/snowplow/scala-stream-collector:0.10.0
+
+# Stream Enrich image
+docker pull snowplow-docker-registry.bintray.io/snowplow/stream-enrich:0.11.0
+
+# Elasticsearch Loader image
+docker pull snowplow-docker-registry.bintray.io/snowplow/elasticsearch-loader:0.10.0
+
+# S3 Loader image
+docker pull snowplow-docker-registry.bintray.io/snowplow/s3-loader:0.6.0
+```
+
+## Building
+
+Alternatively, you can build them yourself:
+
+```bash
+# All images are based on the base image
+docker build -t snowplow/base:0.1.0 base
+
+# Scala Stream Collector image
+docker build -t snowplow/scala-stream-collector:0.10.0 scala-stream-collector/0.10.0
+
+# Stream Enrich image
+docker build -t snowplow/stream-enrich:0.11.0 stream-enrich/0.11.0
+
+# Elasticsearch Loader image
+docker build -t snowplow/elasticsearch-loader:0.10.0 elasticsearch-loader/0.10.0
+
+# S3 Loader image
+docker build -t snowplow/s3-loader:0.6.0 s3-loader/0.6.0
+```
+
+## Running
+
+Create your own config file filling it according to your setup, you can find examples at the
+following locations:
+
+- [Scala Stream Collector configuration][ssc-config]
+- [Stream Enrich configuration][se-config]
+- [Elasticsearch Loader configuration][es-config]
+- [S3 Loader configuration][s3-config]
+
+Next, you can run a container for each component by mounting your configuration directory:
+
+```bash
+# Scala Stream Collector container
+docker run \
+  -v $PWD/scala-stream-collector-config:/snowplow/config \
+  snowplow/scala-stream-collector:0.10.0 \
+  --config /snowplow/config/config.hocon
+
+# Stream Enrich
+docker run \
+  -v $PWD/stream-enrich-config:/snowplow/config \
+  snowplow/stream-enrich:0.11.0 \
+  --config /snowplow/config/config.hocon \
+  --resolver file:/snowplow/config/resolver.json \
+  --enrichments file:/snowplow/config/enrichments/ \
+  --force-ip-lookups-download
+
+# Elasticsearch Loader
+docker run \
+  -v $PWD/elasticsearch-loader-config:/snowplow/config \
+  snowplow/elasticsearch-loader:0.10.0 \
+  --config /snowplow/config/config.hocon
+
+# S3 Loader
+docker run \
+  -v $PWD/s3-loader-config:/snowplow/config \
+  snowplow/s3-loader:0.6.0 \
+  --config /snowplow/config/config.hocon
+```
+
+You can find more information in the readme for each image:
+
+- [Scala Stream Collector readme][ssc-readme]
+- [Stream Enrich readme][se-readme]
+- [Elasticsearch Loader readme][es-readme]
+- [S3 Loader readme][s3-readme]
+
+There is also a Docker Compose example in the [example folder][example].
+
+## Find out more
+
+| Technical Docs             | Setup Guide          | Roadmap & Contributing |
+|----------------------------|----------------------|------------------------|
+| ![i1][techdocs-image]      | ![i2][setup-image]   | ![i3][roadmap-image]   |
+| [Technical Docs][techdocs] | [Setup Guide][setup] | _coming soon_          |
+
+## Copyright and license
+
+Copyright 2017-2017 Snowplow Analytics Ltd.
```

Comment:
Just 2017?

Reply:
This is to make it easier to update in the future, eg. `find . -type f -print0 | xargs -0 sed -i 's/2017-2017/2017-2018/g'`